packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -170,7 +175,7 @@ Autocomplete.propTypes = {
    *
    * Also see: https://github.com/paypal/downshift#itemtostring
    */
-  itemToString: PropTypes.func,
+  itemToString: PropTypes.func.isRequired,
```

Comment:
Since we have a default set for this, we don't need it to be required. This would also mean we could remove `itemToString` from the example components.